src/main/java/pl/sunflux/controllers/rest/DriverController.java

```diff
@@ -0,0 +1,61 @@
+package pl.sunflux.controllers.rest;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.*;
+import pl.sunflux.controllers.rest.containers.driver.ParkingFeeContainer;
+import pl.sunflux.controllers.rest.containers.driver.StartParkingContainer;
+import pl.sunflux.controllers.rest.containers.driver.StopParkingContainer;
+import pl.sunflux.controllers.rest.responses.ValidationError;
+import pl.sunflux.controllers.rest.responses.ValidationErrorBuilder;
+import pl.sunflux.controllers.rest.responses.driver.ParkingFeeResponse;
+import pl.sunflux.controllers.rest.responses.driver.StartParkingResponse;
+import pl.sunflux.controllers.rest.responses.driver.StopParkingResponse;
+import pl.sunflux.parking.ParkingFeeInterface;
+import pl.sunflux.parking.ParkingStartInterface;
+import pl.sunflux.parking.ParkingStopInterface;
+
+import javax.validation.Valid;
+
+@RequestMapping(value = "driver")
+@RestController
+public class DriverController {
+
+    private final ParkingFeeInterface parkingFeeInterface;
```

Comment:
Skąd pomysł na takie rozbicie zasobów? Trochę to utrudnia przeglądanie - trzeba się przebić przez warstwę: fasada, interfejs dla poszczególnego podzasobu, impementację. Jaki widzisz zalety w stosunku do zinline'owania tego w kontrolerze?